src/header.rs

```diff
@@ -5,13 +5,14 @@
 
 use crate::error::Error;
 use byteorder::{BigEndian, ByteOrder};
-use std::io::{Seek, SeekFrom, Read};
+use std::io::{Read, Seek, SeekFrom};
```

Comment:
The same line???

Reply:
This must have happened as a result of the rust formatter on my Vscode, it tries to keep imports in alphabetical order. This can be ignored.